src/functionalTest/java/uk/gov/hmcts/probate/functional/IntegrationTestBase.java

```diff
@@ -20,25 +19,22 @@
     protected TestUtils utils;
 
     protected String orchestratorUrl;
-    protected String submitServiceUrl;
     protected String idamUrl;
 
     @Autowired
     public void initialise(@Value("${probate.orchestrator.service.url}") String orchestratorUrl,
-                           @Value("${probate.submit.url}") String submitServiceUrl,
                            @Value("${user.auth.provider.oauth2.url}") String idamUrl) {
         RestAssured.baseURI = orchestratorUrl;
         RestAssured.defaultParser = Parser.JSON;
         this.orchestratorUrl = orchestratorUrl;
-        this.submitServiceUrl = submitServiceUrl;
         this.idamUrl = idamUrl;
```

Comment:
Are the orchestratorUrl or idamUrl used anywhere else in the code? do they need to be defined here?